content/20.treatments.md

```diff
@@ -617,57 +617,117 @@ Biologics are produced from components of living organisms or viruses. They incl
+TCZ is a humanized monoclonal antibody that binds both to the insoluble and soluble receptor of IL-6, providing de facto inhibition of the IL-6 immune cascade.
 
 **Current Evidence.**
-Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear signs of improvements, according to their results [@url:https://www.genengnews.com/virology/coronavirus/genentech-launches-phase-iii-trial-of-actemra-as-coronavirus-treatment/].
-Also, AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection [@url:https://www.univadis.co.uk/viewarticle/covid-19-italy-launches-an-independent-trial-on-tocilizumab-715741].
-In addition to these independent trials, Roche, in collaboration with the FDA, will start a randomized, double-blind, placebo-controlled phase III trial in early April.
-The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyze its efficiency/safety [@url:https://www.biopharma-reporter.com/Article/2020/03/19/Roche-enters-Phase-III-for-COVID-19-treatment].
-However, previous studies of RA showed that the rate of incident infections in clinical practice patients treated with TCZ was higher than the rate observed during clinical trial [@doi:10.1093/rheumatology/ker223].
-Also, RA patients with chronic hepatitis B (HB) infection showed high risk of HB virus reactivation upon TCZ administration in combination with other RA drugs [@doi:10.1111/1756-185X.13010].
-These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated by the cytokine storm, and preventing the worsening of the clinical picture of the patients by potential new viral infections.
-This aspect should be investigated further in upcoming trials.
-
-Perhaps, the TCZ treatment would best suit patients with severely compromised lungs due to the COVID-19 infection and are therefore at greater risk of death, in order to stop the uncontrolled immune response before it’s too late.
+Tocilizumab is being administered either as a primary or supplemental treatment in {{ebm_tocilizumab_ct}} COVID-19 clinical trials curently registered at clinicaltrials.gov. 
+No randomized, placebo-controlled studies have been released yet for TCZ.
+Therefore, no conclusions can be drawn about its efficacy for the treatment of COVID-19.
+However, early interest in TCZ as a possible treatment for COVID-19 emerged from a very small retrospective study in China that examined 20 patients with severe symptoms in early February 2020 and reported rapid improvement in symptoms following treatment with TCZ [@doi:10.1073/pnas.2005615117].
+Subsequently, a number of restrospective studies have been conducted in several countries.
+<!--
```

Comment:
I was going to add a table demonstrating some of the studies, but then I found that the most recent systematic review already did a better job...